modes/dictation_mode.talon

```diff
@@ -3,12 +3,14 @@ mode: dictation
 ^press <user.keys>$: key("{keys}")
 
 # Everything here should call auto_insert to preserve the state to correctly auto-capitalize/auto-space.
-<user.prose>: auto_insert(prose)
-new line: "\n"
-new paragraph: "\n\n"
-cap <user.word>:
-    result = user.formatted_text(word, "CAPITALIZE_FIRST_WORD")
-    auto_insert(result)
+<user.raw_prose>: auto_insert(raw_prose)
+cap: user.dictation_format_cap()
+# Hyphenated variants are for Dragon.
+(no-caps | no caps): user.dictation_format_no_caps()
+(no-space | no space): user.dictation_format_no_space()
+^cap that$: user.dictation_reformat_cap()
```

Comment:
This comes from Dragon, but I agree there's no reason the inconsistency should live on. I'm now using "cap" consistently everywhere, except when Dragon requires otherwise. It simply refuses to hear "no cap" and will bias to its own "no-caps" (which it prints as "no cap"), so I need to add that or it doesn't work. Tested both modal and modeless versions with Dragon and Conformer.